terraform/modules/agreements-db/variables.tf

```diff
@@ -0,0 +1,17 @@
+variable "organisation" {}
+
+variable "space" {}
+
+variable "environment" {}
+
+variable "postgres_instance_name" {}
+
+variable "postgres_service_plan" {}
+
+variable "cf_username" {
+  #sensitive = true
```

Comment:
Can reinstate these if we're bumping the TF version up to 0.14+ (as discussed)

Reply:
Yep - finally downloaded that TF zip 😴  - updating now